tests/unit/accounts/test_services.py

```diff
@@ -554,32 +560,29 @@ def test_check_totp_value_invalid_totp(self, user_service, monkeypatch):
         assert not valid
         assert limiter.hit.calls == [pretend.call(user.id), pretend.call()]
 
-    @pytest.mark.parametrize(
-        ("challenge", "rp_name", "rp_id"),
-        (["fake_challenge", "fake_rp_name", "fake_rp_id"], [None, None, None]),
```

Comment:
Note: I honestly don't know what purpose the None/None/None served since there's no branch in the method under test.

Reply:
gh pr checkout 10482